Action/vrijbrp.ZgwToVrijbrpEmigratieAction.action.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "ZgwToVrijbrpSOAPEmigratieAction",
+  "$id": "https://vrijbrp.nl/action/vrijbrp.ZgwToVrijbrpSOAPEmigratieAction.action.json",
+  "$schema": "https://json-schema.org/draft/2020-12/action",
+  "version": "0.0.3",
+  "listens": [
+    "vrijbrp.zaak.created"
+  ],
+  "throws": [],
+  "conditions": {
+    "==": [
+      {"var": "object.zaaktype.identificatie"},
+      "B1425"
+    ]
+  },
+  "class": "CommonGateway\\NaamgebruikVrijBRPBundle\\ActionHandler\\ZgwToVrijbrpEmigratieActionHandler",
+  "configuration": {
```

Comment:
location is required in the ActionHandler configuration, so it should be present or not required